thirdeye/thirdeye-frontend/app/pods/components/filter-select/template.hbs

```diff
@@ -1,9 +1,10 @@
-{{#power-select-multiple 
+{{#power-select-multiple
   selected=selectedFilters
   options=viewFilterOptions
   search=(perform searchByFilterName)
   onchange=(action "onFilterChange")
-  placeholder="  Add a filter (Type to search)"
+  searchPlaceholder="Search for a dimension"
+  placeholder="Search for a dimension"
```

Comment:
Since these are the same, let's save this in a variable in the component and pass it in the template

Reply:
fixed. reverted to original